hilti/runtime/include/context.h

```diff
@@ -45,6 +45,12 @@ struct Context {
      */
     resumable::Handle* resumable = nullptr;
 
+    /** The main fiber. */
+    std::unique_ptr<::Fiber> main_fiber;
+
+    /** The currently executing fiber .*/
+    std::unique_ptr<::Fiber>* current_fiber = nullptr;
```

Comment:
Do we want leave this at being raw pointer? The `*` is easy to miss there with the `unique_ptr<T>` part of the same type. (C++ lacks a `std::` thing for a observing, non-managed pointer; that would make it clear here).

Reply:
I missed to clean this up what I removed the typedef for `unique_ptr<::Fiber>` and we don't gain anything by exposing how the `::Fiber` is managed. I'll switch to a raw ptr here and for `Fiber::_caller`.